identity/src/main/java/com/stripe/android/identity/camera/IdentityAggregator.kt

```diff
@@ -24,11 +33,35 @@ internal class IdentityAggregator(
     aggregateResultListener,
     IdentityScanState.Initial(
         type = identityScanType,
-        transitioner = IDDetectorTransitioner(
-            timeoutAt = Clock.markNow() + verificationPage.documentCapture.autocaptureTimeout.milliseconds,
-            iouThreshold = verificationPage.documentCapture.motionBlurMinIou,
-            timeRequired = verificationPage.documentCapture.motionBlurMinDuration
-        )
+        transitioner =
+        if (identityScanType == IdentityScanState.ScanType.SELFIE) {
+            // TODO(ccen) Read params from VerificationPage
+            FaceDetectorTransitioner(
+                VerificationPageStaticContentSelfieCapturePage(
+                    autoCaptureTimeout = 15000,
+                    filePurpose = "selfie",
+                    numSamples = 8,
+                    sampleInterval = 200,
+                    faceDetectorThreshold = 0.8f,
+                    maxCenteredThresholdX = 0.2f,
+                    maxCenteredThresholdY = 0.2f,
+                    minEdgeThreshold = 0.05f,
+                    minCoverageThreshold = 0.07f,
+                    maxCoverageThreshold = 0.8f,
+                    lowResImageMaxDimension = 800,
+                    lowResImageCompressionQuality = 0.82f,
+                    highResImageMaxDimension = 1440,
+                    highResImageCompressionQuality = 0.92f,
+                    highResImageCropPadding = 0.5f,
+                    consentText = "consent"
```

Comment:
Oh these are just now here in order to compile, we're still working on the backend change for `VerificationPageStaticContentSelfieCapturePage`, later this will be parsed from server response